src/bend.ts

```diff
@@ -134,7 +170,7 @@ export class Bend extends Modifier {
     let total_width = 0;
     for (let i = 0; i < this.phrase.length; ++i) {
       const bend = this.phrase[i];
-      if ('width' in bend) {
+      if (bend.width) {
```

Comment:
This is wrong because `bend.width` could be 0.

Reply:
ok, checked with `undefined`